### ISSUE TITLE
Re-tranning model

### ISSUE THREAD

**Issue by 0vulns** [sethuiyer/Image-to-Image-search#10]
I was wondering if I can re-train  `flickr8k_cnn_lstm_v1.p` model

**Comment by sethuiyer** [comment]
Yes, that's definitely in my plans. Make an interface where the user can submit an image and the caption and the model gets re-trained accordingly. That will increase the usability of this by miles.

The model retraining is totally possible.

**Comment by Skyquek** [comment]
What is the method to retrain the model? Where to get the training script?
